.circleci/config.yml

```diff
@@ -66,5 +75,31 @@ workflows:
               only:
                 - master
                 - /\d+\.\d+\.x/
+  build_deploy_releases:
+    jobs:
+      - build:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v\d+\.\d+\.\d+(-rc\.\d+)?$/
+      - deploy:
+          requires:
+            - build
+          filters:
```

Comment:
I don't know if we need to repeat this same filter in every job since they all require (directly or transitively) the `build` job which has the filter.